tools/accuracy_checker/accuracy_checker/postprocessor/nms.py

```diff
@@ -230,3 +230,84 @@ def _nms(self, input_bboxes, input_scores):
             scores *= np.exp(np.negative(np.square(iou_values) / self.sigma))
 
         return np.array(out_ids, dtype=np.int32), np.array(out_scores, dtype=np.float32)
+
+class DIOU_NMS(Postprocessor):
```

Comment:
naming convention does not allow to use underscore in class names
```suggestion
class DIoUNMS(Postprocessor):
```
or
```suggestion
class DistanceIoUNMS(Postprocessor):
```